### PR TITLE
[ja] Localize Event Streaming into Japanese

### LINKED ISSUE #2854: [ja] Localize Event Streaming into Japanese
Localize https://glossary.cncf.io/event-streaming/ into Japanese

### PR DESCRIPTION
**Pull request by katzchang**
### Describe your changes

Localize https://glossary.cncf.io/event-streaming/ into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

Resolves #2854 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ja/event-streaming.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: イベントストリーミング
+status: Completed
+category: 概念
```
Comment (Okabe-Junya) [inline content/ja/event-streaming.md:4]:
```suggestion
category: コンセプト
```

既存の例に合わせて "コンセプト" でお願いいたします

e.g. 

https://github.com/cncf/glossary/blob/5b05c81eedc37e4217c04964ec184a93ebae4e8d/content/ja/agile-software-development.md?plain=1#L4

**Review by Okabe-Junya — CHANGES_REQUESTED** [review]
継続的な貢献ありがとうございます！

Almost LGTMですが、一点だけ細かな修正をお願いいたします

**Review by Okabe-Junya — APPROVED** [review]
LGTMです！

**Review by naonishijima — APPROVED** [review]
LGTM

**Pull request merged**